.gitignore

```diff
@@ -1,14 +1,156 @@
-build/*
-dist/*
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Auto-generated protobuf files
 py/frequenz_api_dispatch.egg-info/*
```

Comment:
This can go, as it is already ignored by other more general rule, but it doesn't hurt to keep it either.